CHANGELOG.md

```diff
@@ -1,8 +1,15 @@
 # Changelog
-## Unreleased
+## [Version 0.2.15](https://github.com/NVISO-BE/ee-outliers/releases/tag/0.2.15) (July 29, 2020)
 ### New features
 - New Sudden Appearance model: the sudden_appearance model looks for outliers by finding the sudden appearance of a 
 certain value of a specific field, or multiple fields
+### Minor changes
+- Updates on dockerfile to enforce security
+- When ee-outliers is unable to write on an index, it return an error log message instead of stopping the program
+- Set "@timestamp" instead of "timestamp" as default timestamps field
+### Bug fixes
+- Fix issue where generated ES dsl queries were not compatible with 7.7.+ version of ES
```

Comment:
- Fixes an issue where generated Elasticsearch DSL queries were not compatible with Elasticsearch version 7.7 and above